_opensearch/query-dsl/term.md

```diff
@@ -7,160 +7,138 @@ nav_order: 30
 
 ```json
+PUT students
 {
-  "took" : 19,
-  "timed_out" : false,
-  "_shards" : {
-    "total" : 1,
-    "successful" : 1,
-    "skipped" : 0,
-    "failed" : 0
-  },
-  "hits" : {
-    "total" : {
-      "value" : 10000,
-      "relation" : "gte"
-    },
-    "max_score" : 17.419369,
-    "hits" : [
-      {
-        "_index" : "shakespeare",
-        "_id" : "34229",
-        "_score" : 17.419369,
-        "_source" : {
-          "type" : "line",
-          "line_id" : 34230,
-          "play_name" : "Hamlet",
-          "speech_number" : 19,
-          "line_number" : "3.1.64",
-          "speaker" : "HAMLET",
-          "text_entry" : "To be, or not to be: that is the question:"
-        }
+  "mappings": {
+    "properties": {
+      "name": {
+        "type": "keyword"
       },
-      {
-        "_index" : "shakespeare",
-        "_id" : "109930",
-        "_score" : 14.883024,
-        "_source" : {
-          "type" : "line",
-          "line_id" : 109931,
-          "play_name" : "A Winters Tale",
-          "speech_number" : 23,
-          "line_number" : "4.4.153",
-          "speaker" : "PERDITA",
-          "text_entry" : "Not like a corse; or if, not to be buried,"
-        }
+      "classes": {
+        "type": "keyword"
       },
-      {
-        "_index" : "shakespeare",
-        "_id" : "103117",
-        "_score" : 14.782743,
-        "_source" : {
-          "type" : "line",
-          "line_id" : 103118,
-          "play_name" : "Twelfth Night",
-          "speech_number" : 53,
-          "line_number" : "1.3.95",
-          "speaker" : "SIR ANDREW",
-          "text_entry" : "will not be seen; or if she be, its four to one"
-        }
+      "min_required": {
+        "type": "integer"
       }
-    ]
+    }
   }
 }
-...
 ```
+{% include copy-curl.html %}
 
-For a list of all full-text queries, see [Full-text queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/).
+Next, index two documents that correspond to students:
 
-If you want to query for an exact term like “HAMLET” in the speaker field and don't need the results to be sorted by relevance scores, a term-level query is more efficient:
+```json
+PUT students/_doc/1
+{
+  "name": "Mary Major",
+  "classes": [ "CS101", "CS102", "MATH101" ],
+  "min_required": 2
+}
+```
+{% include copy-curl.html %}
 
 ```json
-GET shakespeare/_search
+PUT students/_doc/2
+{
+  "name": "John Doe",
+  "classes": [ "CS101", "MATH101", "ENG101" ],
+  "min_required": 2
+}
+```
+{% include copy-curl.html %}
+
+Now search for students who have taken at least two of the following classes: `CS101`, `CS102`, `MATH101`:
+
+```json
+GET students/_search
 {
   "query": {
-    "term": {
-      "speaker": "HAMLET"
+    "terms_set": {
+      "classes": {
+        "terms": [ "CS101", "CS102", "MATH101" ],
+        "minimum_should_match_field": "min_required"
+      }
     }
   }
 }
 ```
+{% include copy-curl.html %}
 
-#### Sample response
+The response contains both students:
```

Comment:
"contains both student names" (the response can't literally contain the students)?